cfme/common/vm_views.py

```diff
@@ -12,6 +12,14 @@
     CheckableManageIQTree, VersionPick, Version, CheckboxSelect, BaseEntitiesView)
 
 
+class NonEditableTable(Table):
+    def fill(self, values):
+        """Clicks on item - fill by selecting required value"""
+
+        value = values.get('name', '<None>')
+        self.row(name=value).click()
```

Comment:
Protocol of `fill` is that if it did any change, it must return True, otherwise False. Also, if this is the "selector" table type, it should probably also reimplement `read` to give you the name of the thing selected :)